### ISSUE TITLE
Frontend Bot Link

### ISSUE THREAD

**Issue by AnthonyOSX** [FeroxRev/Pokemon-Go-Rocket-API#201]
Wondering if you could link my project from here as a UI to this API that non tech people can use. Would be much appreciated, and thanks for the great API!

https://github.com/AnthonyOSX/GO-Bot

**Comment by Krexis** [comment]
Would be great, this stuff is so confusing since Necro forked the projects I cannot figure out how to get the Ferox version to work. It would need google integration first though,

**Comment by HelloKitty** [comment]
No license on your bot. MIT license it and I'll support I guess.

**Comment by jshield** [comment]
We haven't got one either and at this stage it would be a PITA to get everyone who has contributed to signoff

**Comment by HelloKitty** [comment]
@jshield I disagree. People were contributing to an unlicensed work in which the sole creator retains the rights to in most countries. Any license could be added to the current project.

**Comment by jshield** [comment]
@HelloKitty it does indeed work that way, first off those who fork the project are violating Copyright Laws under the Berne Convention because FeroxRev is the only one authorised to make copies and secondly FeroxRev accepting pull requests is violating the copyright of the author of the pull request. This is why Open Source projects that change licenses or add one need to get every contributor to sign a legal document transferring all rights to the project owners.

This is why licenses such as the MIT, BSD and GPL exist in the first place, and why they are referred to as Copyleft Licenses.

**Comment by HelloKitty** [comment]
@jshield That's why FeroxRev is responsible for enforcing copyright violations against his own intellectual property or he risks weakening his rights. The project is unlicensed and therefore the project maintainer can apply any license he'd like unless he has privately entered into licensed agreements with contributors which I double.

A pull request nor contributor has any rights to his code because there is no license and the maintainer effectively has full copyright of the code. Anyone making a pull request is merging changed of copyright protected code that FeroxRev holds the rights to.

An unlicensed Github project, as stated by the VP of Github here http://www.infoworld.com/article/2615869/open-source-software/github-needs-to-take-open-source-seriously.html , leaves the maintainer with sole copyright rights. He would be bound to a license, and all contributors thereafter, if he added one now but right now this is 100% his project. Any fork, pull request is his intellectual property and he'd win in court against you if you tried to sell a bot based on his work right now.

Anyway, MIT license Rocket-API and @AnthonyOSX your bot too and I'd have no problem.

**Comment by jshield** [comment]
It doesn't work that way with respect to individual contributions though as the contributions themselves are not derivatives, you could argue that someone submitting a pull request is de facto giving up ownership but neither the terms of service on github nor any law I have seen clarify it explicitly. Which is why Github tells you when you have more than yourself contributing to a project to pick a license.
see: 

https://help.github.com/articles/open-source-licensing/

In particular: If you want others to use, copy, modify, or contribute back to your project, we strongly encourage you to include an open source license.

This is a very grey area. Can you show actual legal precedent where a patch has been deemed to be solely owned by the projects maintainer and not the author of a patch?

**Comment by jshield** [comment]
by the way I totally agree with the addition of a license. That being said make sure that everyone who has contributed is cool with their code being licensed and if not, have it removed. IMHO is the safest way to proceed.
